nodejs/week1/src/backend/data/meals.json

```diff
@@ -0,0 +1,67 @@
+[
+    {
+      "id": 1,
+      "title": "Indian food in the summer",
+      "maxNumberOfGuests": 5,
+      "description": "A nice night out eating delicious indian food",
+      "createdAt": "2019/12/7 14:34",
+      "price": 67
+    }, 
+  
+      {
+        "id": 2,
+        "title": "barflychickie",
+        "maxNumberOfGuests": 15,
+        "description": "booze with chicken cripsy",
+        "createdAt": "2020/06/06 21:38",
+        "price": 128
+      }, 
+
```

Comment:
Fix formatting. The indentation in these doesn't match the rest of the file.

Reply:
Tip: Set up your text editor with something like prettier. It will automatically format your code everytime you save the file so you don't need to even think about it. 

For example, VSCode has this plugin https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
I have installed the Prettier - Code formatter by Esben Petersen,.,.thanks,.,Future formating will be impeccable